### PR TITLE
feat splash screen app added

### PR DESCRIPTION
**Pull request by abrahamsankutty91**

### CODE REVIEW

**Review by AshwinSampathkumarRK — APPROVED** [review]

File: apps/api/env.development.local [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NODE_ENV=development
```
Comment (muhammadtsneo) [inline apps/api/env.development.local:1]: the file name is incorrect hence gitignore didn't work. missed prefix dot remove the file

File: apps/splash-screen/src/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"use client";
+import * as React from "react";
+
+import { CardType } from "types";
+
+export const Card: React.FC<CardType> = ({ className, children }) => {
+  return (
+    <div
+      className={`${className} p-6 shadow-2xl border border-gray-200 w-[450px] rounded-2xl `}
```
Comment (muhammadtsneo) [inline apps/splash-screen/src/components/Card.tsx:9]: unwanted space? before closing quote

File: apps/splash-screen/src/components/TextField.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+export const TextField: React.FC<TextType> = ({
+  className,
+  label,
+  placeholder,
+}) => {
+  return (
+    <>
+      {label && (
+        <label className=" text-sm font-semibold mb-[10px]" htmlFor={label}>
```
Comment (muhammadtsneo) [inline apps/splash-screen/src/components/TextField.tsx:14]: unwanted space before opening quote

**Review by muhammadtsneo — CHANGES_REQUESTED** [review]
remove the env file from the PR, found extra spaces in class names start and/or end,

File: apps/api/env.development.local (hunk 1 above)
Comment (abrahamsankutty91) [reply apps/api/env.development.local]: fixed

File: apps/splash-screen/src/components/Card.tsx (hunk 2 above)
Comment (abrahamsankutty91) [reply apps/splash-screen/src/components/Card.tsx]: fixed

File: apps/splash-screen/src/components/TextField.tsx (hunk 3 above)
Comment (abrahamsankutty91) [reply apps/splash-screen/src/components/TextField.tsx]: fixed

**Review by muhammadtsneo — APPROVED** [review]

**Pull request merged**